### PR TITLE
Deprecation of Master nomenclature

### PR DESCRIPTION
**Pull request by kaushalmahi12**
**Is your feature request related to a problem? Please provide an existing Issue # , or describe.**
[Master Nomenclature Issue](https://github.com/opensearch-project/performance-analyzer-rca/issues/139)

**Describe the solution you are proposing**
This change is to stop the usage of `master` keyword in the performance-analyzer-rca codebase.

**Additional context**

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/performance-analyzer-rca/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by kkhatua — CHANGES_REQUESTED** [review]
Recommend breaking this into multiple commits spanning groups of packages. 
That will make it easier to split the review across multiple eyes.

File: config/rca_cluster_manager.conf [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "max-flow-units-per-vertex-buffer": 200,
 
   "tags": {
-    "locus": "idle-master-node"
+    "locus": "cluster_manager-node"
```
[the remark this answers is not in the document]
Comment (kaushalmahi12) [reply config/rca_cluster_manager.conf:24]: Yes You are right will make the change.

File: src/main/java/org/opensearch/performanceanalyzer/metrics/AllMetrics.java [hunk 2]
```diff
@@ -789,16 +789,17 @@ public String toString() {
         }
 
         public static class Constants {
-            public static final String PENDING_TASKS_COUNT_VALUE = "Master_PendingQueueSize";
+            public static final String PENDING_TASKS_COUNT_VALUE =
+                    "ClusterManager_PendingQueueSize";
```
[the remark this answers is not in the document]
Comment (kaushalmahi12) [reply src/main/java/org/opensearch/performanceanalyzer/metrics/AllMetrics.java:793]: Ok

File: src/main/java/org/opensearch/performanceanalyzer/metrics/PerformanceAnalyzerMetrics.java [hunk 3]
```diff
@@ -33,17 +33,18 @@ public class PerformanceAnalyzerMetrics {
...
+    public static final String sClusterManagerClusterUpdate = "cluster_manager_cluster_update";
     public static final String sElectionTermPath = "election_term";
     public static final String sHttpPath = "http";
     public static final String sOSPath = "os_metrics";
     public static final String sHeapPath = "heap_metrics";
     public static final String sNodesPath = "node_metrics";
     public static final String sPendingTasksPath = "pending_tasks";
-    public static final String sMasterThrottledTasksPath = "master_throttling_metrics";
+    public static final String sClusterManagerThrottledTasksPath =
+            "cluster_manager_throttling_metrics";
```
[the remark this answers is not in the document]
Comment (kaushalmahi12) [reply src/main/java/org/opensearch/performanceanalyzer/metrics/PerformanceAnalyzerMetrics.java:47]: Ok

File: .github/workflows/gradle.yml [hunk 4]
```diff
@@ -54,8 +54,8 @@ jobs:
     - name: Checkout Performance Analyzer
       uses: actions/checkout@v2
       with:
-        repository: opensearch-project/performance-analyzer
-        ref: main
+        repository: kaushalmahi12/performance-analyzer
```
Comment (dzane17) [inline .github/workflows/gradle.yml:57]: Is this change temporary?

**Comment by kaushalmahi12** [comment]
I will break the commit into smaller ones and raise the PR against those. It will help the reviewers to review the changes without getting tired.

**Pull request closed without merging**